package.json

```diff
@@ -25,35 +25,28 @@
         "ocamlfind",
         "sexplib"
       ]
-    },
-    "buildsInSource": "_build"
```

Comment:
Could you briefly say why this is not needed?

Reply:
If you use Dune, then Dune has support for esy's out of source package builds, which means that esy can speed things up during local development. By default `buildsInSource` is assumed to be `true`, and you can set it to `false` or `_build`. If you use `false` it copies inefficiently but works with everything all the time. If you use `_build` it builds into `_build` with a small perf hit. If your build system supports out of source builds, you leave it as the default `true` and get the best perf.